doc/source/_templates/layout.html

```diff
@@ -1,25 +1,145 @@
 {% extends "!layout.html" %}
 
 {%- block extrahead %}
+<!-- Was unable to link to a _static stylesheet. Depending on the type of
+page, the relative path wouldn't always go to the same directory. -->
```

Comment:
Something like this?
```suggestion
<link rel="stylesheet" href="{{ pathto('_static/numpy.css', 1) }}" type="text/css" />
```
from https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/layout.html#L24

Reply:
Works perfectly -- thank you! Having the css in the template meant having to rebuild every time I made a style change.